lib/webpack-helpers/create-webpack-config.js

```diff
@@ -44,16 +44,23 @@ function createWebpackConfig(urc) {
     urc.jsEntry
   ].filter(Boolean);
 
+  const entry = {
+    main: mainEntry,
+    vendor: [
+      resolvePkg('react', { cwd: jsEntryDir }),
+      resolvePkg('react-dom', { cwd: jsEntryDir }),
```

Comment:
I understand the intent here is to allow for an app without React. Will these `resolvePkg` calls safely return null or undefined and not loudly error?

Reply:
I tried running underreact on a blank project, it simply returns `undefined`.